src/assets/styles.css

```diff
@@ -252,9 +252,12 @@ polygon.rs-logo-shape,
   color: darkred;
 }
 
+.rs-box-error {
+  display: flex;
```

Comment:
We should explicitly define the `flex-direction` here.

Reply:
Direction `row` is the default, but you're right. Better be safe than sorry.